editor/editor_settings_dialog.cpp

```diff
@@ -314,18 +314,22 @@ void EditorSettingsDialog::_event_config_confirmed() {
 
 void EditorSettingsDialog::_update_builtin_action(const String &p_name, const Array &p_events) {
 	Array old_input_array = EditorSettings::get_singleton()->get_builtin_action_overrides(p_name);
+	if (old_input_array.is_empty()) {
+		List<Ref<InputEvent>> defaults = InputMap::get_singleton()->get_builtins_with_feature_overrides_applied()[current_edited_identifier];
```

Comment:
It would be nice to have a cheaper way to find the defaults, instead of building a HashMap and fetching one element.